src/Avalonia.DesignerSupport/Remote/Stubs.cs

```diff
@@ -195,6 +195,7 @@ class ClipboardStub : IClipboard
     class CursorFactoryStub : IStandardCursorFactory
     {
         public IPlatformHandle GetCursor(StandardCursorType cursorType) => new PlatformHandle(IntPtr.Zero, "STUB");
+        public IPlatformHandle CreateCursor(IBitmapImpl cursor, PixelPoint hotSpot) => new PlatformHandle(IntPtr.Zero, "STUB");
```

Comment:
Should we return an `ICursorImpl` here instead of a simple `IPlatformHandle`?

Reply:
Yes. Also cursor lifetime management will be a thing now